library/std/src/f128.rs

```diff
@@ -22,7 +22,10 @@ impl f128 {
     ///
     /// ```
     /// #![feature(f128)]
-    /// # #[cfg(reliable_f128_math)] {
+    /// # #![cfg_attr(not(bootstrap), feature(cfg_target_has_reliable_f16_f128))]
+    /// # #![cfg_attr(not(bootstrap), expect(internal_features))]
```

Comment:
To bad that `rustdoc` [`#![doc(test(attr(...)))]`](https://doc.rust-lang.org/rustdoc/write-documentation/the-doc-attribute.html#testattr) attribute only works at the crate-root level (and not module level), otherwise we could have saved us a lot of redundancy by just having in each module:
```rust
#![doc(test(attr(feature(cfg_target_has_reliable_f16_f128))))]
#![doc(test(attr(expect(internal_features))))]
```

Reply:
Oh that would be so nice. Agreed, tests are too noisy with config, but at least they shouldn't need to be updated very frequently.